### PR TITLE
Robel Tekeste_Solution

### PR DESCRIPTION
**Pull request by Robizzy27**
I can't seem to get the links to change color depending on wether they're active or not. Please advise.

Thanks
Rob

### CODE REVIEW

**Comment by Robizzy27** [comment]
Comfort Score: 3
Completeness: 4

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+body {
```
Comment (nolds9) [inline stylesheet.css]: In the future, be careful with your code's indentation. WDI style guide recommends one `tab` indent for content after each rule declaration, i.e. 
```
body {
  background: lemonchiffron;
}
```

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+img{
+float: right;
+border: 2px solid white;
+}
+a{
+font-family: fantasy;
+color: #00CC00;
+font-weight: bold;
+}
+a.active{
```
Comment (nolds9) [inline stylesheet.css]: Would recommend looking into css pseudo-selectors for more targeted stylings. i.e. `a:hover { color: red; }` to make set the text color to red after any hover's over a link. [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes)

**Comment by nolds9** [comment]
:+1:  Great job on this assignment. In the future feel free to leave questions or areas of code where you want specific feedback.

**Pull request closed without merging**